server/config/config.go

```diff
@@ -440,6 +446,10 @@ func (c *Config) Adjust(meta *toml.MetaData, reloading bool) error {
 		return err
 	}
 
+	if c.TSOMaxIndex != 0 && c.TSOMaxIndex <= c.TSOUniqueIndex {
```

Comment:
Should we check the`TSOMaxIndex` as well to ensure it won't be too big to occupy too many logical bits?